### ISSUE TITLE
bug: `leap.nvim`'s beacons no longer show with the recent `leap.nvim` updates

### ISSUE THREAD

**Issue by echaya** [vscode-neovim/vscode-neovim#2145]
### Check the following:

- [X] I have checked the behavior after setting "vscode-neovim.neovimClean" in VSCode settings.
- [X] I have read the vscode-neovim docs.
- [X] I have searched existing issues.

### Neovim version (nvim -v)

0.9.5

### Operating system/version

Win 10

### Describe the bug

Not sure if related to https://github.com/ggandor/leap.nvim/commit/c507733b04af71aefe6f025e9f5b53cc5560f757, but `leap.nvim`'s beacons (those highlighted labels) are no longer showing since I have updated the package yesterday. 

### Steps To Reproduce

pressing `swh` should highlight all `wh` from "when" with those beacons/labels, but nothing is highlighted at the moment

```JSON
    {
        "key": "ctrl+alt+k",
        "command": "-bookmarks.toggle",
        "when": "editorTextFocus"
    },
    {
        "key": "alt+a",
        "command": "jupyter.insertCellAbove",
        "when": "editorTextFocus && jupyter.hascodecells && !notebookEditorFocused"
    },
    {
        "key": "ctrl+; a",
        "command": "-jupyter.insertCellAbove",
        "when": "editorTextFocus && jupyter.hascodecells && !notebookEditorFocused"
    },

```

![latest leap on vscode](https://github.com/vscode-neovim/vscode-neovim/assets/34254408/5720426b-d870-47d0-aaa2-8a1d20bea80e)

### Expected Behavior

![latest leap on nvim](https://github.com/vscode-neovim/vscode-neovim/assets/34254408/8c096c5e-090c-4654-90ea-a487c3bfa34b)

**Comment by xiyaowong** [comment]
Shouldn't this be reported to leap.nvim? I checked the recent commits of leap.nvim, and they have modified the default values for highlights. You can try customizing the related highlights in leap to see if this issue can be resolved. For limitations on highlights in vscode-neovim, refer to: https://github.com/vscode-neovim/vscode-neovim#-highlights

**Comment by echaya** [comment]
> Shouldn't this be reported to leap.nvim? I checked the recent commits of leap.nvim, and they have modified the default values for highlights. You can try customizing the related highlights in leap to see if this issue can be resolved. For limitations on highlights in vscode-neovim, refer to: https://github.com/vscode-neovim/vscode-neovim#-highlights

why I should report to leap.nvim please? As far as I know, it is not broken in nvim but vscode-neovim. And based my test, It is not related to color.

@xiyaowong PLEASE REFRAIN FROM CLOSING ANY ISSUE that you don't know how to fix or simply you don't care. 

While I appreciate your contribute to this community, it is evident that you have been closing numerous issues without actually tackling anything. This is nothing but unhealthy for the repo/community.

@theol0403 @asvetliakov  I'm more than happy to sponsor this project should such activity not happening again. 
 


@

**Comment by xiyaowong** [comment]
@echaya Please speak based on facts. I have closed a total of two issues you raised, each with an explanation and examples, all of which I have personally tested.

**Comment by echaya** [comment]
#2125 :  was :perfect tested: it and that's why you said https://github.com/vscode-neovim/vscode-neovim/issues/2115#issuecomment-2177797492 7 posts after you closed the issue. And up to today, I dont know how to apply https://github.com/vscode-neovim/vscode-neovim/issues/2115#issuecomment-2177424967. 
#2142: was closed with same example no one asked
Now this, the 3rd time.

**Comment by xiyaowong** [comment]
https://github.com/vscode-neovim/vscode-neovim/issues/2142#issuecomment-2211686219

> because 'compositeKeys' and keybinding have no relation to each other. Moreover
> 'when' in VSCode is not accessible.

Aren't these two explanations enough? I could have closed that issue directly, but I still took the time to provide examples!

**Comment by echaya** [comment]
> [#2142 (comment)](https://github.com/vscode-neovim/vscode-neovim/issues/2142#issuecomment-2211686219)
> 
> > because 'compositeKeys' and keybinding have no relation to each other. Moreover
> > 'when' in VSCode is not accessible.
> 
> Aren't these two explanations enough? I could have closed that issue directly, but I still took the time to provide examples!

Isn't it a perfect example how you closed an issue without taking the time to study the issue and providing incorrect / not-so-useful suggestions. :)

Just searched a bit on the first page of the closed issues, glad that I'm not the only one. Sure you think you close it legitimately. 

#2146 closed leaving other people guessing what he wrote 
#2123 closed without proving suggestion / context. 

@theol0403 @asvetliakov @ollien @windwp @justinmk

**Comment by xiyaowong** [comment]
有意思吗？你列举的这几个issue有什么说服力？不搜索已有的issue难道不能关了？质问没做某件事不本身就是一个建议？我也发现你挺有意思，对我有意见

**Comment by xiyaowong** [comment]
那我问你，我给你提的建议你有亲自尝试吗？我中午没睡觉特意尝试了，这个leap的问题就是高亮的问题，你自己试了吗？

**Comment by xiyaowong** [comment]
甩一个issue出来，然后我们费劲心思给你解答，结果你还不满意！后面你的issue我不管了

**Comment by theol0403** [comment]
@echaya thank you for reporting this issue.

Please remember you are not entitled to customer support. Myself and all the other contributors do this in our free time and can choose what we want to work on. The issue tracker, ultimately, represents actionable items (TODOs) for the dev team to address. Additionally, issues are being tackled much more swiftly than in the past (when I took over, there were 300+ unanswered issues). This is partly due to a ruthless curbing of redundant, actionless, and low-quality issues. Your report was not one of these, but aggressively cleaning up addressed issues is important when managing a popular open-source project.

That being said, I understand the frustration when you take the time to write a report and it gets dismissed with little apparent regard, and I apologize if it comes across as hostile. 

However, most issues mentioned in this debate (including this issue) were not closed in error:
* https://github.com/vscode-neovim/vscode-neovim/issues/2142: the answer is correct, it is impossible with vscode api
* https://github.com/vscode-neovim/vscode-neovim/issues/2146: this is a fundamental limitation in the 2-way sync, and can be improved by the config flag linked
* https://github.com/vscode-neovim/vscode-neovim/issues/2123: was a low-quality issue with no actionable items or reproducible steps
* this issue: mentioned in the readme, you cannot link to built-in highlight groups because they are ignored to prevent `syntax on` highlight clashes. leap.nvim released an update which caused this change; the fix must be applied to leap.nvim. 

Additionally, @xiyaowong has made more practical PRs to close issues than anyone else, so the criticism is unneeded. 

TLDR; closing issues after a provided (albeit brief) solution is not bad management but is instead required for efficient management for busy volunteer developers. 

@xiyaowong please remember you represent the project and the work that has been done by every dev before you. It is counter-productive to be defensive and hostile. Make sure you treat issues and the community with respect, even if you believe some issues are redundant. 

Thanks everyone
